docs/source/compute/show-gpus-all.txt

```diff
@@ -0,0 +1,61 @@
+COMMON_GPU  AVAILABLE_QUANTITIES  
```

Comment:
Does this need to be committed to the repo? Wouldn't build.sh generate this on the fly for rtd? If not, is it possible to configure it to do that?

Reply:
It doesn't do it on the fly, likely due to requiring cloud credentials.